test/strategy/rysk/RyskStrategy.fork.spec.ts

```diff
@@ -355,9 +355,55 @@ describe('Rysk Strategy (mainnet fork tests)', () => {
       expect(await usdc.balanceOf(vault.address)).to.gte('999999000');
       expect(await strategy.investedAssets()).to.gte('1186057000');
     });
-  });
 
-  describe('more complex scenario...?', async () => {});
+    it('correctly summs amounts for pending deposit, pending withdrawal, unredeemed and redeemed shares ', async () => {
+      // scenario:
+      // 1. invest 1000
+      // 2. epoch++
+      // 3. shares increase 50%
+      // 4. initiate withdraw for 500
+      // 5. epoch++
+      // 6. complete withdraw and get 750 in the vault
```

Comment:
so the initial ratio of shares to underlying is 1 to 1?

Reply:
I think 1 share > 1 underlying here, but that's not relevant for this test since the numbers only refer to the underlying (USDC) amounts, so this should hold true regardless of the price per share. I changed the [comment](https://github.com/lindy-labs/sc_solidity-contracts/pull/216/commits/fec0fc34a03f3e28b4cf8ed548b8720d607c61ec) to make this more obvious